java-sdk/radar-catalog-server/build.gradle.kts

```diff
@@ -14,3 +21,7 @@ dependencies {
 application {
     mainClass.set("org.radarbase.schema.service.SourceCatalogueServer")
 }
+
+radarKotlin {
+    sentryEnabled.set(true)
```

Comment:
javaVersion.set(Versions.java)
    log4j2Version.set(Versions.log4j2)
    
    not needed?

Reply:
@ewelinagr No, defaults are defined in the radar-commons kotlin project gradle plugin. Better to use these defaults.